Board/src/main/java/com/example/board/member/MemberService.java

```diff
@@ -0,0 +1,9 @@
+package com.example.board.member;
+
+public interface MemberService{
+
+    MemberDto createMember(MemberDto memberDto);
+    MemberDto updateMember(MemberDto memberDto);
+    RuntimeException deleteMember(String name);
```

Comment:
반환타입이 RuntimeException는 아닌 것 같아요  😭
```suggestion
    void deleteMember(String name);
```